moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectWebsite.js

```diff
@@ -97,9 +97,15 @@ const ProjectSummaryProjectWebsite = ({
               id="moped-project-website"
               label={null}
               onChange={handleProjectWebsiteChange}
-              value={website} />
+              value={website}
+              error={!isWebsiteValid}
```

Comment:
would you also add the `helperText` prop here? it can say "Website is not a valid URL" when there's an error.

Reply:
Perfect - I will add that! 🙏